packages/block-editor/src/store/reducer.js

```diff
@@ -1711,6 +1711,21 @@ export function lastBlockInserted( state = {}, action ) {
 	return state;
 }
 
+/**
+ * Reducer returning the html node of the iframed editor's wrapper.
+ *
+ * @param {Object|null} state  Current state.
+ * @param {Object}      action Dispatched action.
+ *
+ * @return {Object|null} Updated state.
+ */
+export function iframedEditorCanvasWrapper( state = null, action ) {
+	if ( action.type === 'SET_IFRAMED_EDITOR_CANVAS_WRAPPER' ) {
+		return action.node;
+	}
+	return state;
+}
+
```

Comment:
I can't help but wonder if there is a more appropriate way to share the node ref from the iframe to other components.
I don't have an answer, this is just a side note about something that kinda surprised me. 😄

Reply:
Im not sure.  🤷‍♀️   If I could prop drill it down to the block-popover in this same package we wouldn't need the store.  We are transferring this cross-package by props from block-popover from block-editor package to popover from the component package, the store at this point is just the shortcut to allow block-popover to pass that as a prop.

Alternatively, we could not pass the prop... and use refs that are already in the popover component to do a scoped query selection to get that actual `.editor-styles-wrapper`, but that seems a bit biased and ad-hoc for code in the components package so it seemed having another prop made more sense 🤔

Reply:
Yeah it's probably not a big deal, and something we can iterate on if folks think it's a bit icky.

Reply:
We can remove all the store stuff - 169bcbf - by making a little pseudo store with an object for reference.  I think it beats inflating the actual block-editor store unnecessarily.  Unfortunately it doesn't seem like the tree structure allows us to leverage any props or context from the iframe component for this.

Reply:
Wait! I think I've found the ref to rule them all!

**`__unstableContentRef`**

It's documented [here](https://github.com/WordPress/gutenberg/blob/641060d92e2723d82144ae1206828db7e19d3cf4/packages/block-editor/src/components/block-tools/index.js#L30), and readily available in `BlockPopover`!

I've literally just tried this, and it seems to work for me on a few quick tests!
```js
// block-popover.js

return (
  <Popover
    // ...many props
    __unstableEditorCanvasWrapper={ __unstableContentRef.current }
  >
    // ...content stuff
  </Popover>
);
```

Would you mind double checking if it misses some edge cases you might have investigated already while working on this PR? 🙇‍♂️

Reply:
Great catch!  That does it!  That makes sense now that I see how its set up.

There is another contentRef in the popover component itself, but it refers to the content of the actual popover itself and not the editor.  I breezed right past this contentRef assuming it was that!